ai-ml/t5-model-serving/client-app/src/app.py

```diff
@@ -40,15 +40,18 @@ def text_to_text_function(text: str = "Hello World", from_lang=LANG_MAP, to_lang
   resp = requests.post(PREDICTION_URL, json=payload, headers=headers)
   if resp.status_code == 200:
     content = resp.json()
-    return content.get("text")
+    resp = content.get("text")
   else:
-    return "Oops, something went wrong!"
+    resp = "Oops, something went wrong!"
+
+  return resp
```

Comment:
This code change doesn't change the flow, no?

Reply:
The flow stayed the same except `text_to_text_function` should have only one return line at the end of the function. The function `text_to_text_function` is a callback passed to the FastDash constructor that calls [_infer_variable_names](https://github.com/dkedar7/fast_dash/blob/9a58f2d13cc8e578cdd1ee460f28d54ffc4bee74/fast_dash/utils.py#L391). As result `_infer_variable_names` is a root cause of this change.